.github/workflows/PR.check.yml

```diff
@@ -0,0 +1,31 @@
+name: PR. Build
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
may be need change to master/develop
if it actually need I can try change to use Project/Reposiory variable for it

Reply:
Great, thanks!
Yeah, let's just change to master, develop (master branch = latest release, while most relevant commits will be in develop)

Reply:
I’ve changed triggers, build will run for prs to any branch